scripts/populate_preprint_providers.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""Populate development database with Preprint Provicer elements"""
+
+import logging
+
+from framework.transactions.context import TokuTransaction
+from website.app import init_app
+from website.models import PreprintProvider
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
+
+
+def update_or_create(provider_data):
+    provider = PreprintProvider.load(provider_data['_id'])
+    if provider:
+        for key, val in provider_data.iteritems():
+            setattr(provider, key, val)
+        changed_fields = provider.save()
+        if changed_fields:
+            print('Updated {}: {}'.format(provider.name, changed_fields))
+        return provider, False
+    else:
+        new_provider = PreprintProvider(**provider_data)
+        new_provider.save()
+        provider = PreprintProvider.load(new_provider._id)
+        print('Added new preprint provider: {}'.format(provider._id))
+        return new_provider, True
+
+
+def main():
+    PREPRINT_PROVIDERS = [
+        {
+            '_id': 'socarxiv',
+            'name': 'SocArXiv',
+            'description': 'Open archive of the social sciences',
+            'banner_name': 'socarxiv-banner.png',
+            'logo_name': 'socarxiv-logo.png',
```

Comment:
These should include a key for `external_url`.

I just discovered that none of the external PrePriPros have valid SSL certs, so `http://socarxiv.org` is fine for now.